bodhi/server/models.py

```diff
@@ -61,6 +61,133 @@
 except ImportError:
     log.warning("Could not import 'rpm'")
```

Comment:
If I put the first try block here then can you please tell me where this try block should go:https://github.com/ankit01ojha/bodhi/blob/8f4a951731fc8a2b32439092ae3587a404a3e076/bodhi/server/models.py#L59-L62   ?